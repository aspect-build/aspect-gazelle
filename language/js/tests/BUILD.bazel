load("@aspect_gazelle//:gazelle.bzl", "gazelle_generation_test")
load("@gazelle//:def.bzl", "gazelle_binary")

# gazelle:ignore

# The gazelle binary to run local gazelle tests.
# Internal only for tests.
gazelle_binary(
    name = "gazelle_js_test_binary",
    testonly = True,
    languages = [
        "@gazelle//language/proto:go_default_library",
        "//:js",
    ],
    visibility = ["//visibility:private"],
)

# A separate generation test for each test case
[
    gazelle_generation_test(
        name = "%s_test" % test_workspace.replace("/WORKSPACE", ""),
        dir = test_workspace.replace("/WORKSPACE", ""),
        gazelle_binary = ":gazelle_js_test_binary",
    )
    for test_workspace in glob(["*/WORKSPACE"])
]
