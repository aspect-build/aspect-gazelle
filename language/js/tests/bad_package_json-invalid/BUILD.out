# gazelle:js_npm_package enabled
